test/calculator.test.mjs

```diff
@@ -51,94 +51,128 @@ describe('calculator', function () {
+    assert.ok(depressedOperatorKey, 'operator key is depressed');
   })
-
-  xit('should only allow one operator key to be depressed at a time', function () {
+  it('should only allow one operator key to be depressed at a time', function () {
     // write your own tests
-    assert.strictEqual(true, false, 'update this test');
+    button.multiply.click();
+    button.add.click();
+    button.divide.click();
+    button.multiply.click();
+    const depressedOperatorKey = document.querySelectorAll('.is-depressed')
+    //key.classList.add('is-depressed'); // Operator keys should be depressed when they're clicked on
+    assert.strictEqual(depressedOperatorKey.length, 1, 'Only one operator key can be pressed consecutively');
   })
 
-  xit('should show new number if pressed after an operator key', function () {
+  it('should show new number if pressed after an operator key', function () {
     // write your own tests
-    assert.strictEqual(true, false, 'update this test');
+    button.one.click()
+    button.add.click()
+    button.four.click()
+    console.log(display.textContent)
+    assert.strict(display.textContent, 4, 'expected number should equal 4');
   });
 
-  xit('should add two numbers together', function () {
+  it('should add two numbers together', function () {
     // write your own tests
     button.nine.click();
     button.add.click();
     button.eight.click();
     button.calculate.click();
+    console.log(display.textContent)
     assert.equal(display.textContent,  17)
   })
-  xit('should subtract two numbers', function () {
+  it('should subtract two numbers', function () {
     // write your own tests
     button.nine.click();
     button.subtract.click();
     button.eight.click();
     button.calculate.click();
+    console.log(display.textContent)
     assert.equal(display.textContent,  1)
   })
-  xit('should multiply two numbers', function () {
+  it('should multiply two numbers', function () {
     // write your own tests
     button.four.click();
     button.multiply.click();
     button.seven.click();
     button.calculate.click();
+    console.log(display.textContent)
     assert.equal(display.textContent, 28)
   })
-  xit('should divide two numbers', function () {
+  it('should divide two numbers', function () {
     // write your own tests
     button.four.click();
     button.four.click();
     button.divide.click();
     button.one.click();
     button.one.click();
     button.calculate.click();
+    console.log(display.textContent)
     assert.equal(display.textContent, 4)
   })
   // write more tests below. Think about how to test for "unhappy paths".
   // Try to break it in as many ways as you can!
-
-  xit('can chain multiple operations together', function () {
+  xit('can only add one decimal at a time', function () {
+    button.four.click();
+    button.decimal.click();
+    button.decimal.click();
+    const depressedDecimal = document.querySelector('is-depressed');
```

Comment:
There are a few things causing issues here. It looks like you are trying to find all elements with a class of `is-depressed`. In order to do that you need to use `querySelectorAll`. That will return a nodelist (kinda like an array) of all elements that match the selector. `querySelector` will just return the first element that matches. But you have a typo in your selector. For classes you need a `.` before the classname. So it should actually be `document.querySelectorAll('.is-depressed')` 

But that actually isn't what you are testing. `is-depressed` is the class applied the operator button elements when selected, not all the buttons. So it only works for the add, subtract, multiply and divide buttons. It's just used to style them to look like they are depressed. What you really want to test is if there is ever more than one `.` in the display. So really you should check `display.textContent` and verify there is only one decimal even after clicking the decimal button a few times. There are a few ways to check for the number of characters in a string but here's simple way I found on stack overflow

`assert.strictEqual(display.textContent.split('.').length - 1, 1, 'only 1 decimal can be clicked')`

What this is doing is taking the display string and splitting it up into an array with every instance of `.` being the delimiter or split point. then you find the length and subtract by 1 to get the amount of delimiters found.